source/stylesheets/_refills-header.scss

```diff
@@ -88,14 +88,33 @@
     @include refills-inset;
     border-radius: 30px;
     color: darken(desaturate($refills-logo-color, 50%), 15%);
-    display: inline;
+    display: inline-block;
     font-family: $refills-narrow-sans;
     font-size: 0.8em;
     font-weight: 400;
+    margin-bottom: 4em;
     padding: 0.5em 1.4em;
 
     @include media($refills-large-screen) {
       font-size: 0.8em;
     }
   }
+
+  .empties-banner {
+    $empties-color: #776faa;
+    background-color: $empties-color;
+    border-radius: 5px;
+    border: 1px solid darken(transparentize($empties-color, 0.1), 10%);
+    box-shadow: 0 2px 0px darken(transparentize($empties-color, 0.7), 10%);
+    color: lighten(#c8c0ff, 8);
+    display: inline-block;
+    font-family: $refills-narrow-sans;
+    padding: 1em 2em;
+
+    &:hover,
+    &:focus {
+      background-color: lighten($empties-color, 1%);
+      color: lighten(#c8c0ff, 10);
```

Comment:
Color literals like `#c8c0ff` should only be used in variable declarations; they should be referred to via variable everywhere else.